UMN-AutoPackager/public/Get-NewPackageFile.ps1

```diff
@@ -0,0 +1,139 @@
+function Get-NewPackageFile {
+<#
+    .SYNOPSIS
+    Cmdlet that saves a file from a file path, UNC, or URI and saves it to the specified location.
+    .DESCRIPTION
+    Get-NewPackageFile is a function that when provided with a Source file path, UNC, or URI (as a string) and a Destination path or
+    UNC (as a string) will copy or download the Source file to the Destination. Characters of a space (" ") or an escaped space ("%20")
+    are replaced with an underscore ("_") when saved to the Destination. Returns object information of the saved file.
+    .PARAMETER Source
+    A string that must be a file path, UNC, or URI of the file to be saved to the Destination location. Wildcards are not permitted.
+    .PARAMETER Destination
+    A string that must be a file path or UNC where the Source file will be saved. Wildcards are not permitted.
+    .EXAMPLE
+    Get-NewPackageFile -Source "C:\Test\File1.txt" -Destination "D:\Temp\File2.txt"
+    .EXAMPLE
+    Get-NewPackageFile -Source "C:\Test\File1.txt" -Destination "D:\Temp\"
+    .EXAMPLE
+    Get-NewPackageFile -Source "C:\Test\File1.txt" -Destination "\\fileserver\Temp\File1.txt"
+    .EXAMPLE
+    Get-NewPackageFile -Source "\\fileserver\Test\File1.txt" -Destination "D:\Temp\File1.txt"
+    .EXAMPLE
+    Get-NewPackageFile -Source "\\fileserver1\Test\File1.txt" -Destination "\\fileserver2\Temp\File1.txt"
+    .EXAMPLE
+    Get-NewPackageFile -Source "https://webserver.xyz/Test/File1.txt" -Destination "C:\Temp\File1.txt"
+    .EXAMPLE
+    Get-NewPackageFile -Source "https://webserver.xyz/Test/File1.txt" -Destination "\\fileserver\Temp\File1.txt"
+    .INPUTS
+    String
+    .OUTPUTS
+    System.IO.FileInfo
+    Get-NewPackageFile returns information on the file saved in the location specified in the Destination parameter.
+    .NOTES
+    If the Destination parameter is only a file path and does not include a filename, the filename obtained from the Source parameter
+    is used during the copy. Requires use of the "Get-RedirectedUri" function.
+#>
```

Comment:
See help comment on other function, we should standardize on this being above the function not inside.